src/libcollections/btree/map.rs

```diff
@@ -276,6 +344,14 @@ pub struct OccupiedEntry<'a, K: 'a, V: 'a> {
 
 impl<K: Ord, V> BTreeMap<K, V> {
     /// Makes a new empty BTreeMap with a reasonable choice for B.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
Not a problem at all - plan to contribute a lot more going forward, so have a whole bunch of PRs to come. Best to learn the right way to do these things now!

Fixed!